minecraft_servers/NinjaMC/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "server_name": "NinjaMC",
```

Comment:
server_name please in lowercase (folder name, too!)

Reply:
i changed it